CONTRIBUTING.md

```diff
@@ -53,24 +53,32 @@ For other ways to use `tldrl`, such as linting an entire directory, check out (w
 [`tldr tldrl`](https://github.com/tldr-pages/tldr/blob/master/pages/common/tldrl.md)
 
 ### Token syntax
-User-provided values should use the `{{token}}` syntax in order to allow clients
-to highlight them. 
+User-provided values should use the `{{token}}` syntax
+in order to allow clients to highlight them.
+Use [`snake_case`](https://en.wikipedia.org/wiki/Snake_case)
+for multi-word tokens.
 
-Some examples: 
-- `tar cf {{file}}`
-- `ln -s {{path/to/original/file}} {{path/to/link}}`
-- `mysql {{database_name}}`
-- `unrar x {{compressed.rar}}`
+Keep the following guidelines in mind when writing the tokens:
+
+- If the example is clearer with an actual value rather than a generic placeholder, use the actual value. For example - use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.
+
+- For any reference to a path or folder, use `{{path/to/<placeholder>}}`. For example - `ln -s {{path/to/file}} {{path/to/symlink}}`. In case of a possible reference both to a file or a folder, use `{{path/to/file_or_folder}}`
```

Comment:
How about `For any reference to a path or folder` --> `For references to paths to files or folders`?